sdk/appcenter/src/main/java/com/microsoft/appcenter/AppCenter.java

```diff
@@ -190,6 +190,11 @@
      */
     private DefaultAppCenterFuture<Boolean> mSetMaxStorageSizeFuture;
 
+    /**
+     *
```

Comment:
Missing description.

Reply:
Yep, this is just placeholder/reminder to add 😄